src/main/java/com/synopsys/integration/jenkins/scan/ScanPipelineCommands.java

```diff
@@ -72,7 +74,9 @@ public int runScanner(Map<String, Object> scanParameters) throws ScannerJenkinsE
         }
 
         try {
-            if (exitCode != 0) {
+            if (exitCodeToMessage.containsKey(exitCode)) {
+                throw new NoStackTraceException(exitCodeToMessage.get(exitCode));
```

Comment:
Need to check if we can use the **`Run`** object here from `createPipelineCommand` method to set current build as failed or aborted.

```
run.getPreviousBuild().setResult(Result.FAILURE);
run.getNextBuild().setResult(Result.FAILURE);
```

We need to check which method (`getPreviousBuild()` or `getNextBuild()`) get the current build.

Also need to check whether we need to use `Result.FAILURE` or `Result.ABORT`